src/datasources/cache/__tests__/fake.cache.service.ts

```diff
@@ -32,6 +33,11 @@ export class FakeCacheService implements ICacheService, ICacheReadiness {
     return Promise.resolve(1);
   }
 
+  getCounter(key: string): Promise<number | null> {
+    const value = this.cache[key];
+    return isNumber(value) ? Promise.resolve(value) : Promise.resolve(null);
```

Comment:
Nit: if we expect this to only be an integer, we can remove reliance on lodash and use `Number.isInteger` instead.

Reply:
True, thanks! Done in [a295546](https://github.com/safe-global/safe-client-gateway/pull/1953/commits/a2955464350166bb48010ca21df49066a2ff27cd) (I had to add an extra check as `Number.isInteger` does not narrow the type).